pages/blog/page/[page_index].tsx

```diff
@@ -17,7 +17,7 @@ import getMarkdownData from '../../../utils/github/getMarkdownData'
 import { useCMS } from 'tinacms'
 import OpenAuthoringSiteForm from '../../../components/layout/OpenAuthoringSiteForm'
 import { useForm } from 'tinacms'
-import { withOpenAuthoringErrorHandler } from '../../../open-authoring/withOpenAuthoringErrorHandler'
+import { withOpenAuthoringErrorHandler } from '../../../open-authoring/errors/withOpenAuthoringErrorHandler'
```

Comment:
Could the behaviour provided by `withOpenAuthoringErrorHandler` be added to `OpenAuthoringProvider`?

If it can, then you wouldn't need to add this to each page manually

Reply:
Not in a way I can think of. The AuthProvider is used outside of each page context, and the withOpenAuthoringErrorHandler uses the "previewError" prop passed from getStaticProps for the page view

Reply:
Looks like Next passes those to the `_app` inside of `props.pageProps` so you could pass it like so:

```
  <OpenAuthoring error={props.pageProps.previewError}>
    ...
```